### ISSUE TITLE
First load of docs site does not work if opened in a background tab

### ISSUE THREAD

**Issue by todduk** [angular/angular#39153]
# 📚 Docs or angular.io bug report

### Description
The first time I open the docs site in a background tab, in Chrome, the document body does not display.

## 🔬 Minimal Reproduction
0. Ensure you have not visited the docs page in question recently.
1. Browse a site with a link to the docs site, you can use this link; https://angular.io/api/common/NgIf
1. Use middle mouse click to open the docs link in a background tab
1. Wait a minute or more
1. Focus the docs tab, the document body has not loaded

### What's the affected URL?**
https://angular.io/api/common/NgIf
etc.

## 📷Screenshot
![image](https://user-images.githubusercontent.com/1766553/95313422-9c266c80-08ec-11eb-8d9f-a13d9dd16e0e.png)

## 🔥 Exception or Error
![image](https://user-images.githubusercontent.com/1766553/95313570-c2e4a300-08ec-11eb-9f82-6588b146cb99.png)

## 🌍  Your Environment
Win 10

### Browser info
Chrome Version 85.0.4183.102 (Official Build) (64-bit)

**Comment by gkalpak** [comment]
This is a duplicate of #28114. Let's track the problem there.

**tl;dr**
It is not 100% clear what is causing this (although there are some theories), but in v11 we'll have a way to work around it by means of #36847. (See #36847 for more details.)
